server/@types/GoalType.ts

```diff
@@ -7,8 +7,6 @@ export type Goal = {
   completedDate?: string
   targetDate: string
   goalOrder?: number
```

Comment:
this is now non-nullable in the backend so can remove `?`

Reply:
updated